org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -763,6 +786,9 @@ private void asyncRenderComponents(MDashboardContent dashboardContent, Dashboard
 			}
     		int thisClientId = Env.getAD_Client_ID(Env.getCtx());
     		if((thisClientId == 0 && systemAccess) || thisClientId != 0) {
+    			if(m_WindowNo >= 0) {
+    				addDrillAcrossEventListener(AD_Process_ID);
+    			}
```

Comment:
why the need of the m_WindowNo ? Isn't it always needed to add the drill across listener ?

Reply:
If I'm not wrong, the Drill Assistant can be opened only if we send WindowNo to it (I'm sending it at [line 1011](https://github.com/idempiere/idempiere/pull/1734/files/0639c7091b03abfd821dfe36c42bd00086c84147#diff-7e90c79eeb3052a98a51cf4e2b513402eb62c80a838279445b0a01affc53361dR1011) )

Reply:
I means you can reduce the changes by removing the use of m_windowno and just use 0. no need to add m_windowno instance variable and no need add a new render method either.

Reply:
I see, ok